console_conf/controllers/identity.py

```diff
@@ -116,10 +117,21 @@ def write_login_details(fp, username, ips):
         )
     else:
         first_ip = ips[0]
+        key_info = None
```

Comment:
Other code calls `host_key_info()`, so the logic here should be moved to there instead of only handling the confined version in some cases.